syncd/VendorSai.cpp

```diff
@@ -716,7 +722,13 @@ sai_status_t VendorSai::bulkRemove(
 
         default:
             SWSS_LOG_ERROR("not implemented, FIXME");
-            return SAI_STATUS_FAILURE;
+            return SAI_STATUS_NOT_IMPLEMENTED;
+    }
+
+    if (!ptr)
+    {
+        SWSS_LOG_ERROR("remove bulk not supported from SAI, object_type = %u", object_type);
```

Comment:
instead of %u, use string and serialize_object_type to get string representation